models/articles.model.js

```diff
@@ -0,0 +1,6 @@
+const db = require("../db/connection")
+exports.fetchArticles = () => {
+    return db.query("SELECT * FROM articles ").then((articles) => {
+        return articles.rows
+    })
+}
```

Comment:
You should have a separate function for fetching an article by ID rather than using fetchArticles.